### DISCUSSION TITLE
Pybricks offline

### DISCUSSION THREAD

**Discussion by Jimmy106-e** [pybricks/support#727] [Q&A]
Hello, i’ve seen that i’m the docs it says that a can install pybricks offline but i don’t know how.
Does anyone know how to?

**Comment by laurensvalk** [comment]
You can use Pybricks Code offline without additional steps. If you turn off your internet connection, the site should still load.

If you prefer to use Pybricks Code in its own window (instead of a browser tab), you can click "install as app" in the settings menu:

![image](https://user-images.githubusercontent.com/12326241/192265600-1bb3e8c0-8551-4710-8be2-2c014a0f11bb.png)

**Reply by Jimmy106-e** [reply]
The install as app doesn’t show up
![8A090F6B-56EF-4820-ADD3-01F4FED294A3](https://user-images.githubusercontent.com/113257494/192265886-4ddb4c8b-dabb-433a-8a24-5893e7493faa.jpeg)

**Reply by laurensvalk** [reply]
The button goes away if it is already installed as an app. Could you try searching among your apps to confirm it hasn't already been installed?

**Reply by Jimmy106-e** [reply]
I cant find it

**Reply by Jimmy106-e** [reply]
Can you give me a file (just asking)?

**Reply by laurensvalk** [reply]
What do you see if you type `chrome://apps/` in the address bar of your browser?

Do you see Pybricks there? If so, right-click and select 'Remove from Chrome'.

Then you may be able to install it as an app again.
